specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2020-03-01/examples/Apps_CheckNameAvailability.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-03-01",
```

Comment:
Let's discuss this new version internally.

Reply:
ok, sure